examples/gamma_hierarchy/gamma_gamma_hier.h

```diff
@@ -5,8 +5,8 @@
 #include <src/hierarchies/base_hierarchy.h>
 #include <src/hierarchies/conjugate_hierarchy.h>
```

Comment:
Please note that the Gamma-Gamma example does not currently compile because of this inclusion